### ISSUE TITLE
Get DOI for citation

### ISSUE THREAD

**Issue by jlandmann** [GLAMOS/dataflow#18]
My opinion is that as soon as the database has included all the data we need for the beginning, we should get it a DOI and then make it citeable.

**Comment by yweidmann** [comment]
Agree. Basically everything is setup for the usage of DOI (in collaboration with the ETH library). But we have to clarify the format of the exported database and the periodicity of the exports / versions (e.g. early at summer).
